mocks/mocks.go

```diff
@@ -0,0 +1,92 @@
+package mock
+
+import (
+	"net/http"
+
+	"github.com/blacklightcms/go-recurly/recurly"
+)
+
+var _ recurly.Transactions = &MockTransactionsService{}
+
+// MockTransactionsService mocks the transaction service.
+type MockTransactionsService struct {
```

Comment:
Fixed in [bbaea8](https://github.com/blacklightcms/go-recurly/pull/10/commits/fad5453852cf15bd32cdadb74bfe5677a4bbaea8)